src/routes/user.js

```diff
@@ -7,6 +7,46 @@ const User = require("../models/user")
 
 const USER_SAFE_DETAILS = ["firstName", "lastName", "age", "gender", "about", "photoUrl", "skills"]
 
+route.get("/feed", userAuth, async (req, res) => {
+    try {
+        const user = req.user
+        const page = parseInt(req.query.page) || 1
+        let limit = parseInt(req.query.limit) || 10
+        limit = limit > 100 ? 100 : limit
+
+        const skip = (page - 1) * limit
+        
+        const requests = await ConnectionRequest.find({
+            $or: [
+                { fromUserId: user._id },
+                { toUserId: user._id }
+            ]
+        }).select(["fromUserId", "toUserId"])
```

Comment:
.select() is used to filter out required data instead of sending irrelevant data